images-list

```diff
@@ -30,6 +30,6 @@ quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.13.4
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.13.4
 quay.io/calico/ctl rancher/calico-ctl v3.13.4
 us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-external-dns v0.6.0
-quay.io/open-policy-agent/gatekeeper rancher/opa-gatekeeper v3.1.0-beta.8
+openpolicyagent/gatekeeper rancher/opa-gatekeeper v3.1.0-beta.11
```

Comment:
does it need the `quay.io`?

Reply:
@deniseschannon No, looks like they moved from quay.io to dockerhub since the latest tag they have in quay.io is `v3.1.0-beta.9`